opentracing/logfields.py

```diff
@@ -0,0 +1,52 @@
+# Copyright The OpenTracing Authors
```

Comment:
what if we called this module "logs.py"? Then we could add helper methods like `logs.log_error(span, exc_type, exc_val, exc_tb)` (which btw is currently logged with non-standard fields: https://github.com/opentracing/opentracing-python/pull/99/files#diff-994e6f1956eaad29d5b74c46af16bf3aR217